W-13839595-rtf-mule-cadence-kt (#1087)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/runtime-patch-updates.adoc

| Rollback 
include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=RTF4]
| Restart apps
include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=RTF5]
| Keep running applications 
include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=RTF6]
| End of Extended Support 
include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=RTF7]
| Retirement  
include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=RTF8]
| Supportability 
include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=RTF9]
|===

include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=MuleIntro3]

=== Runtime Fabric

include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=MuleIntro4]

Runtime Fabric requires 
include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=MuleIntro5]

include::release-notes::partial$mule-runtime/mule-lts-edge-release-cadence.adoc[tags=MuleIntro6]

== Disable Autoupgrades

> Starting Mule runtime engine 4.5.0, autoupgrades are disabled automatically. Any Mule app deployment through the API without specifying the build tag goes through successfully and the latest build is auto selected. The grace period section is only valid for versions < 4.5.0. With the version 4.5.0+, all builds under a Mule runtime engine version is available for deployment.

Reviewer: ```suggestion
Starting with Mule runtime engine 4.5.0, autoupgrades are disabled automatically. If a Mule app is successfully deployed through the API without a build tag, the latest build is auto selected. The grace period section is only valid for versions earlier than 4.5.0. With versions later than 4.5.0, all builds under a Mule runtime engine version are available for deployment.
```

Reviewer: Is the grace period section available for versions earlier than 4.5.0? If not, the text should be "The grace period section is available only for versions earlier than 4.5.0."
